docs/spec/rum_v3_network.json

```diff
@@ -0,0 +1,48 @@
+{
+    "$id": "docs/spec/rum_v3_network.json",
+    "title": "Network",
+    "description": "Network related info.",
+    "type": [
+        "object",
+        "null"
+    ],
+    "properties": {
+        "et": {
+            "description": "effective_type",
+            "type": [
+                "string",
+                "null"
+            ],
+            "maxLength": 1024
+        },
+        "rtt": {
+            "description": "round trip time",
+            "type": [
+                "integer",
+                "null"
+            ]
+        },
+        "dlk": {
+            "description": "downlink",
+            "type": [
+                "number",
+                "null"
+            ]
+        },
+        "dlkm": {
+            "description": "downlink_max",
+            "type": [
+                "number",
+                "null"
+            ]
+        },
+        "ph": {
```

Comment:
To what field, this maps to the Network Info API?

Reply:
short for "physical layer", equivalent to "type".